pkg/token/execCredentialPlugin.go

```diff
@@ -29,6 +31,14 @@ type execCredentialPlugin struct {
 }
 
 func New(o *Options) (ExecCredentialPlugin, error) {
+	env := os.Getenv(execInfoEnv)
+	var execCredential clientauthentication.ExecCredential
+	if error := json.Unmarshal([]byte(env), &execCredential); error != nil {
```

Comment:
`error` is a type so generally you don't use the type name as variable name. conventionally, `err` is used

Reply:
Updated with err for next commit. Thanks!